tests/functionTests/completionTests_fish.txt

```diff
@@ -64,6 +64,7 @@ Possible Candidates:
 127.0.0.1 examples.StreamingRpcs        (uni- and bi-directional streams)
 127.0.0.1 examples.StatusHandling                   (gRPC error handling)
 /^127.0.0.1 grpc.reflection.*$
+/^127.0.0.1 grpc.reflection.*$
```

Comment:
in latest version, grpc server adds two reflection services (one stable and one alpha)